iiblib/iibclient.py

```diff
@@ -303,20 +303,24 @@ def make_auth(self, iib_session):
 class IIBSession(object):
     """Helper class to support iib requests and authentication"""
 
-    def __init__(self, hostname, retries=3, verify=True):
+    def __init__(self, hostname, retries=3, verify=True, backoff_factor=2):
```

Comment:
How did you choose this factor value? If I'm not mistaken then the request will wait once for 4 seconds. Is that helpful?

Reply:
It will sleep for {backoff factor} * (2 ** ({number of total retries} - 1)) seconds between tries. SO the waiting time is increased after each try.
We discussed it with @midnightercz and agreed on this number.

Reply:
My question is, what problem does it solve? Are micro outages common in IIB?

Reply:
I guess so, maybe we can agree of higher value. Right now it waits 3, 6, 12 seconds between each retry. It means 21 seconds from first one will be the last retry. 
I would go for time about 60 or more seconds in total.

Reply:
Isn't it 0, 4 ,8?

Reply:
yeah, just trying if you pay attention